### PR TITLE
Restaurant panel

### PR DESCRIPTION
**Pull request by kashafYaseen**
CRUD operations with views for restaurant panel

### CODE REVIEW

File: app/controllers/vouchers_controller.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class VouchersController < ApplicationController
   def index
-    @voucher = Voucher.all
+    @q = Voucher.ransack(params[:q])
+    @pagy, @voucher =   pagy(@q.result(distinct: true), items: params[:per_page])
```
Comment (Sultan0099) [inline app/controllers/vouchers_controller.rb]: remove extra space after =, and make sure to use the plural name for arrays or any object which you think has more than one record e.g @voucher to @vouchers.

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module ApplicationHelper
+  include Pagy::Frontend
+end
```
Comment (Sultan0099) [inline app/helpers/application_helper.rb]: Add line at the end of file.

File: app/views/customer/_customer_table.html.erb [hunk 3]
```diff
@@ -10,14 +10,26 @@
...
 
     <div class="card-body">
-      <table id="datatablesSimple">
+      <div class="row">
+        <div class="col-6">
+          <%= form_with url: "/customer", method: :get do |form| %>
+            entries per page
+            <%= select_tag "per_page", options_for_select([5,10,20,50], :selected=>params[:per_page]),  { onchange: "this.form.submit();"} %>
+          <% end %>
+        </div>
```
Comment (Sultan0099) [inline app/views/customer/_customer_table.html.erb:19]: Add an empty line after multiline tags.

File: app/views/deals/index.html.erb [hunk 4]
```diff
@@ -7,7 +7,7 @@
         <main>
           <%= render 'employee/card'%>
           <div class="container-fluid px-4">
-            <%= render 'deals/dealTable'%>
+            <%= render 'deal_table'%>
```
Comment (Sultan0099) [inline app/views/deals/index.html.erb]: Add space after 'deal_table'.

File: app/views/discounts/_search_form.html.erb [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<%= search_form_for @q, url:discounts_index_url do |f| %>
```
Comment (Sultan0099) [inline app/views/discounts/_search_form.html.erb:1]: Use remote to submit the search form.

File: app/views/food_items/_item_table.html.erb [hunk 6]
```diff
@@ -11,10 +11,19 @@
     </div>
 
     <div class="card-body">
-      <table id="datatablesSimple">
+      <div class="row">
```
Comment (Sultan0099) [inline app/views/food_items/_item_table.html.erb:14]: Correct the indentations.

File: app/views/reviews/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,17 @@
+<% if current_employee%>
```
Comment (Sultan0099) [inline app/views/reviews/index.html.erb]: Correct indentations

File: app/views/reviews/_table_form.html.erb [hunk 8]
```diff
@@ -0,0 +1,13 @@
+<% @review.each do |review| %>
```
Comment (Sultan0099) [inline app/views/reviews/_table_form.html.erb]: Can you please explain why you are not getting orders from review?

Comment (kashafYaseen) [reply app/views/reviews/_table_form.html.erb]: This is the reviews controller and I am getting the reviews directly so, there is no need of getting orders that's why i didn't get orders from reviews.

**Pull request merged**